lib/u3d/unity_versions.rb

```diff
@@ -114,6 +114,8 @@ module UnityVersions
     UNITY_BETAS = 'https://unity3d.com/unity/beta/archive'.freeze
```

Comment:
if this is broken fully, then either we document this isn't used (and change the content) or remove the field and bump the minor version of u3d.